packages/react-components/react-positioning/src/usePositioning.ts

```diff
@@ -267,17 +267,21 @@ function useContainerRef(updatePosition: () => void, enabled: boolean) {
       Object.assign(container.style, { position: 'fixed', left: 0, top: 0, margin: 0 });
     }
 
-    toggleScrollListener(container, prevContainer, updatePosition);
-
-    updatePosition();
+    // toggleScrollListener requires computed styles; thus use RAF to prevent forced style reevaluation.
+    window.requestAnimationFrame(() => {
```

Comment:
We need to use a proper `window` instance, otherwise `requestAnimationFrame` may not be triggered properly.

You can get it from `useFluent()` (it's already used in this component):

```ts
const { targetDocument } = useFluent();

targetDocument?.defaultView?.requestAnimationFrame()
```